common/OpTestSystem.py

```diff
@@ -1008,7 +1008,7 @@ def sys_ipmi_boot_system_to_petitboot(self, i_console):
 
         # Exiting to petitboot shell
         self.console.expect('Petitboot', timeout=BMC_CONST.PETITBOOT_TIMEOUT)
-        time.sleep(20)
+        self.console.expect('x=exit', timeout=10)
```

Comment:
@sammj - is there a way we can wait for pb-discover to finish its mojo?

For things requiring disks/network we may have a bit of fun working that out in a reliable way.